### ISSUE TITLE
The return of error

### ISSUE THREAD

**Issue by mdeweerd** [Dolibarr/dolibarr#28319]
The return of error
```plaintext
_pagehead() (void)
But there is no more such function. All _pagehead are now return float|int
```

_Originally posted by @eldy in https://github.com/Dolibarr/dolibarr/issues/28315#issuecomment-1954852971_
            

Was it gone?  It's back for a reason:

https://github.com/Dolibarr/dolibarr/blob/b343181789a1cc278d8baa9882c3cfbf71594aaf/htdocs/core/modules/product/doc/pdf_standard.modules.php#L706

**Comment by mdeweerd** [comment]
Also look at:
```
htdocs\core\modules\facture\doc\pdf_sponge.modules.php:2051 PhanPluginUnknownArrayMethodReturnType Method \pdf_sponge::_pagehead() has a return type of array, but does not specify any key types or value types
```

The type is different there as well  .

**Comment by eldy** [comment]
> ```
>           The return of error
> ```
> 
> ```
> _pagehead() (void)
> But there is no more such function. All _pagehead are now return float|int
> ```
> 
> _Originally posted by @eldy in [#28315 (comment)](https://github.com/Dolibarr/dolibarr/pull/28315#issuecomment-1954852971)_
> 
> Was it gone? It's back for a reason:
> 
> https://github.com/Dolibarr/dolibarr/blob/b343181789a1cc278d8baa9882c3cfbf71594aaf/htdocs/core/modules/product/doc/pdf_standard.modules.php#L706

I forgot this one... Fixed

**Comment by mdeweerd** [comment]
And htdocs\core\modules\facture\doc\pdf_sponge.modules.php !

**Comment by mdeweerd** [comment]
Fixed in ca7f06f .
